lib/service/unit.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+# Wrapper for a formula to handle service-related stuff like parsing and
+# generating the plist file.
+module Service
+  class Unit
```

Comment:
Not huge on this name. Any other suggestions?

Reply:
I had `Service::Service` before, but that's not much better.

Reply:
`Service::Definition` or `Service::FormulaWrapper` or something?